payjoin-test-utils/src/lib.rs

```diff
@@ -52,8 +52,7 @@ impl TestServices {
         let redis = init_redis();
         let db_host = format!("127.0.0.1:{}", redis.0);
         let directory = init_directory(db_host, cert_key.clone()).await?;
-        let gateway_origin =
-            http::Uri::from_str(&format!("https://localhost:{}", directory.0)).unwrap();
+        let gateway_origin = Uri::from_str(&format!("https://localhost:{}", directory.0)).unwrap();
```

Comment:
Looks like a straggling `.unwrap()` that could be replaced with `?`

Or is this an unrelated change?